### ISSUE TITLE
Provide accessible tooltips instead of relying on title attribute

### ISSUE THREAD

**Issue by kimisgold** [omeka/omeka-s#2405]
Right now we rely on `title` attributes to get a native hover tooltip, but [there are many accessibility concerns with that practice](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/title#accessibility_concerns). This is an issue for tracking research and development for custom tooltips within Omeka S. 

[The W3C has an issue for developing an official example](https://github.com/w3c/aria-practices/issues/127), and [this seems to be the latest working version](https://zoebijl.github.io/apg-tooltip/). It was posted in 2022, though, and hasn't yet been published to the APG.

**Comment by aviannamiller** [comment]
Adding a few examples / implementations as points of reference here:

- [Technology Services at Illinois - Accessibility Examples: ARIA Tooltip](https://techservicesillinois.github.io/accessibility/aria-examples/tooltip.html): Overall a good model, though the tooltip presents on hover but dismisses upon moving pointer. It will only remain with pointer movement over the tooltip content upon the trigger element gaining keyboard focus or a pointer click (according to [SC 1.4.13: Content on Hover or Focus](https://www.w3.org/TR/WCAG22/#content-on-hover-or-focus), appearance on hover should be enough to keep the tooltip displayed when the pointer moves into the additional content).
- [Internet Archive Item Page](https://archive.org/details/clevelandart-1981.7-amulet-seal-in-the-f): Tooltips display to provide more details on actions buttons and file types for download (for some reason, seemingly no screen reader announcement on the latter only). Overall a good implementation, though same issue as above with the tooltip failing to be hoverable for users operating via pointer. Also, not noticing a way to dismiss the tooltip without moving focus, and at times, they do obscure content (otherwise, this might be okay).
- [Europeana Item Page](https://www.europeana.eu/en/item/91619/SMVK_EM_objekt_2417737): This is a pretty nice implementation, hoverable and persistent, though I don't see any apparent mechanism to dismiss without moving focus / hover, and content is not consistently communicated to screen readers.
- [JSTOR Image Browse](https://www.jstor.org/images): Info tooltip at the top of the main content area is dismissible, hoverable, and persistent. With VO, at least, the text content can only be accessed in a sort of roundabout way (through opening the More Content menu), but it is ultimately accessible to screen readers.

**Comment by kimisgold** [comment]
~~I've got a couple of branches going. The original `accessible-tooltip` branch has a solution based on the wip example from WCAG. It includes some positioning calculations, but runs into issues with z-index overlap with the main content container as well as the open search options panel.~~

~~The `accessible-tooltip-popover` branch has one based on the [popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API). It gets around many of those z-index issues from the other branch. I've also drafted a modal version of the search options form in this branch, which introduced a new issue with popovers persisting when focus has moved: in this case, the tooltip for the search options button persists even when focus has moved to the open modal.~~

~~Another thing with popover is that it's got some funky positioning. It seems like it can only be positioned relative to the viewport unless you use the [new CSS anchor positioning, which has only been recently supported](https://caniuse.com/css-anchor-positioning). I'm betting @aviannamiller is going to see more awkwardness, as her Safari is capped at 17.6.~~

([Reposted for Classic here.](https://github.com/omeka/Omeka/issues/1061#issuecomment-3884930897))

**Comment by aviannamiller** [comment]
Getting the following when I try to take a look on `accessible-tooltip`:
```
InvalidArgumentException: Could not find file css/tooltips.css! in /var/www/html/amiller/Classic/application/libraries/globals.php:1316
Stack trace:
#0 /var/www/html/amiller/Classic/application/libraries/globals.php(1275): web_path_to('css/tooltips.cs...')
#1 /var/www/html/amiller/Classic/application/libraries/globals.php(1221): src('css/tooltips.cs...', 'css', 'css', '3.2')
#2 /var/www/html/amiller/Classic/application/libraries/globals.php(1102): css_src('tooltips', 'css', '3.2')
#3 /var/www/html/amiller/Classic/application/libraries/globals.php(1098): queue_css_file('tooltips', 'all', false, 'css', '3.2')
#4 /var/www/html/amiller/Classic/admin/themes/default/common/header.php(16): queue_css_file(Array)
#5 /var/www/html/amiller/Classic/application/libraries/Omeka/View.php(114): include('/var/www/html/a...')
#6 /var/www/html/amiller/Classic/application/libraries/Zend/View/Abstract.php(889): Omeka_View->_run('/var/www/html/a...')
#7 /var/www/html/amiller/Classic/application/libraries/Zend/View/Helper/Partial.php(109): Zend_View_Abstract->render(NULL)
#8 /var/www/html/amiller/Classic/application/libraries/Zend/View/Abstract.php(350): Zend_View_Helper_Partial->partial('common/header.p...', Array)
#9 /var/www/html/amiller/Classic/application/libraries/globals.php(1701): Zend_View_Abstract->__call('partial', Array)
#10 /var/www/html/amiller/Classic/application/libraries/globals.php(1715): common('header', Array)
#11 /var/www/html/amiller/Classic/admin/themes/default/index.php(3): head(Array)
#12 /var/www/html/amiller/Classic/application/libraries/Omeka/View.php(114): include('/var/www/html/a...')
#13 /var/www/html/amiller/Classic/application/libraries/Zend/View/Abstract.php(889): Omeka_View->_run('/var/www/html/a...')
#14 /var/www/html/amiller/Classic/application/libraries/Zend/Controller/Action/Helper/ViewRenderer.php(912): Zend_View_Abstract->render(NULL)
#15 /var/www/html/amiller/Classic/application/controllers/IndexController.php(16): Zend_Controller_Action_Helper_ViewRenderer->renderScript('index.php')
#16 /var/www/html/amiller/Classic/application/libraries/Zend/Controller/Action.php(516): IndexController->indexAction()
#17 /var/www/html/amiller/Classic/application/libraries/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('indexAction')
#18 /var/www/html/amiller/Classic/application/libraries/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#19 /var/www/html/amiller/Classic/application/libraries/Zend/Application/Bootstrap/Bootstrap.php(106): Zend_Controller_Front->dispatch()
#20 /var/www/html/amiller/Classic/application/libraries/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#21 /var/www/html/amiller/Classic/application/libraries/Omeka/Application.php(73): Zend_Application->run()
#22 /var/www/html/amiller/Classic/admin/index.php(28): Omeka_Application->run()
#23 {main}
```

**Comment by kimisgold** [comment]
My mistake, I was mixing up my issues across platforms. That comment was supposed to go in https://github.com/omeka/Omeka/issues/1061. I'll repost it there.
